### PR TITLE
[UI] Pass remote provider page titles verbatim

### PR DESCRIPTION
**Pull request by humblenginr**
Create a function to convert **"Meshmap"** to **"MeshMap"**

Screenshot 2021-07-16 at 3 14 22 PM

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Does this need to be dynamic or can the text from the remote provider be used directly?

**Comment by sudo-NithishKarthik** [comment]
@dhruv0000  had a better suggestion 🤔 ,i'll get in touch with him and update this ✌🏽

**Comment by sudo-NithishKarthik** [comment]
Now it is totally dynamic 😃

File: ui/pages/extension/[component].js [hunk 1]
```diff
@@ -40,29 +42,30 @@ function capitalize(string) {
...
-  * converts "Meshmap" to "MeshMap"
-  *
-  * @param {string} string
-  *
-  * @returns {string}
-*/
-const capitalizeMeshmap = (string) => string == "Meshmap" ? "MeshMap" : string;
 
-const getComponentName = compose(capitalizeMeshmap,capitalize,extractComponentName,getPath);
+// const getComponentName = compose(capitalizeMeshmap,capitalize,extractComponentName,getPath);
```
Comment (leecalcote) [inline ui/pages/extension/[component].js:46]: Perhaps delete this line?

**Review by leecalcote — DISMISSED** [review]

File: ui/pages/extension/[component].js (hunk 1 above)
Comment (leecalcote) [reply ui/pages/extension/[component].js:46]:
```suggestion
```

**Pull request merged**